### ISSUE TITLE
no login possible after update

### ISSUE THREAD

**Issue by nicogithub22** [globaleaks/globaleaks-whistleblowing-software#4299]
### What version of GlobaLeaks are you using?

I was using 4.14.8 and updatet with 
These commands:
```
wget https://deb.globaleaks.org/install-globaleaks.sh
chmod +x install-globaleaks.sh
./install-globaleaks.sh
```

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

After I executed the above script, Globaleaks was updated but when I try to log in to the Admin GUI page nothing happens. 
I am not logged in. 
But I do not get an error message, wrong password for example.
Maybe someone has a hint?

### Proposed solution

-reboot ubuntu system
-tried different browsers and users

**Comment by evilaliv3** [comment]
Thank you @nicogithub22 

Actually the update from release 4 to 5 does not include differences in the backend so it should not have any difference in the authentication process.

Do you have screenshots or videos where you can show the malfunction?

What if you try to recover your access using the account recovery keys?

**Comment by evilaliv3** [comment]
@nicogithub22 : do you have any update?

**Comment by evilaliv3** [comment]
Thank you @nicogithub22 

would you please install release 5.0.25 and retry with the password reset procedure using the account recovery key?

**Comment by nicogithub22** [comment]
Thank you @evilaliv3 

Unfortunately, the update did not bring about any change. 
I still cannot log in to the frontend of Globaleaks. No matter what I try, normal login or forgotten password. There is always only a short loading circle and nothing more.

**Comment by evilaliv3** [comment]
Thank you @nicogithub22 

This is actually very strange. 
If you could pass me a link to the platform i may give it a look.

**Comment by nicogithub22** [comment]
Yes, very strange for me too.
Here is the LINK: meldestelle.gett-group.com 
I can also give you my username and password, then you can also try with it, it doesn't work at the moment anyway.

Thanks a lot for your time.

**Comment by evilaliv3** [comment]
You are welcome.

Actually i see that your setup features an Apache proxy. I tried to perform a login with wrong credentials and a password reset. Both fails because the proxy reply on behalf of globaleaks blocking the connections to globaleaks.

What is the reason of this proxy server?

**Comment by nicogithub22** [comment]
This is because we run a VM with Linux internally and make it publicly available via our firewall.
We have not changed anything in the design.

**Comment by evilaliv3** [comment]
If you try to look at the network console of your browser, you will se that apache is blocking the connections to globaleaks.

It's not globaleaks.

**Comment by nicogithub22** [comment]
ah, I've seen it. I'll have a look in the firewall tomorrow.

**Comment by nicogithub22** [comment]
I did not find it in the firewall. Perhaps something has been changed in your version. Regarding HTTP headers?

Can you do anything with the error message?
![2024-11-14 11_11_39-Christian Herr _ GETT Gerätetechnik GmbH _ n minkley@gett-group com _ Microsoft ](https://github.com/user-attachments/assets/8132e836-6249-4cd4-8061-ae9e84f56d47)

Otherwise would there be a possibility to make a backup of the data and import it into a new instance?

**Comment by nicogithub22** [comment]
Hello, is it possibel to backup data and import it into a new instance?

**Comment by evilaliv3** [comment]
Yes you could do by following the common backup and restore procedure: https://docs.globaleaks.org/en/stable/user/admin/BackupAndRestore.html

You basically need to copy the directory /var/globaleaks to a new system.

**Comment by nicogithub22** [comment]
Unfortunately this did not work.
I have completely reinstalled the software
